contracts/zxp/game/seasons/mechanics/StakerRewards.sol

```diff
@@ -49,6 +51,7 @@ contract StakerRewards is ObjectRegistryClient, IStakerRewards {
     function onUnstake(
         uint id,
         address to,
+        // why don't we check this value against a mapping in GameVault ?

```

Comment:
because this is a Season object, it doesn't actually know about the game vault. it is not on the same registry as it.